test/built-ins/BigInt/asIntN/length.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2017 Josh Wolfe. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: pending
+description: BigInt.asIntN.length descriptor
+info: >
+  BigInt.asIntN ( bits, bigint )
+
+  17 ECMAScript Standard Built-in Objects
+
+includes: [propertyHelper.js]
+---*/
+
+assert.sameValue(BigInt.asIntN.length, 2);
+
+verifyNotEnumerable(BigInt.asIntN, 'length');
+verifyNotWritable(BigInt.asIntN, 'length');
+verifyConfigurable(BigInt.asIntN, 'length');
```

Comment:
we're trying to get rid of these separate verify helpers. `verifyProperty` should be fine:

```js
verifyProperty(BigInt.asIntN, 'length', {
  value: 2,
  enumerable: false,
  writable: false,
  configurable: true
});
```

The same applies for other test files in this PR